### PR TITLE
Move the spec from the FedCM repo

### PR DESCRIPTION
**Pull request by samuelgoto**
Move the files created in this PR:

https://github.com/w3c-fedid/FedCM/issues/641

Once this gets merged, I'll delete the files from the FedCM repo.

### CODE REVIEW

File: spec/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.PHONY: all spec idl clean
```
Comment (cbiesinger) [inline spec/Makefile:1]: should you update the toplevel makefile to also build this?

File: spec/index.bs [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+Default Biblio Display: inline
+
+Text Macro: FALSE <code>false</code>
+Text Macro: TRUE <code>true</code>
+Text Macro: RP Relying Party
+Text Macro: IDP Identity Provider
+
+Abstract: A Web Platform API that allows websites to notify the browser when their users login and logout.
+
+Test Suite: https://github.com/web-platform-tests/wpt/tree/master/fedcm
```
Comment (cbiesinger) [inline spec/index.bs]: add /fedcm-login-status here (sorry, missed this in the initial review)

**Review by cbiesinger — APPROVED** [review]
lgtm assuming you verified this builds without warnings

File: spec/index.bs (hunk 2 above)
Comment (samuelgoto) [reply spec/index.bs]: done

File: spec/Makefile (hunk 1 above)
Comment (samuelgoto) [reply spec/Makefile:1]: yep, i'll kick off another PR so that i can make this PR clean.

**Pull request merged**